lib/tag-list/style.scss

```diff
@@ -20,19 +20,21 @@
     flex: none;
     justify-content: space-between;
     align-items: center;
-    padding: 5px 17px 2px;
+    padding: 0 17px 13px;
+    border-bottom: 1px solid;
 
     h2 {
-      font-size: 14px;
-      font-weight: 500;
+      font-size: 16px;
+      font-weight: 400;
       margin-bottom: 0;
       text-transform: capitalize;
+      color: $studio-gray-80;
```

Comment:
Annoying that the panel title component is hard-coded as `theme-fg-dim`.

Instead of adding colors to override, let's remove the `PanelTitle` usage and just make it a normal `<h2>` in the tags list:

`<h2 class="theme-color-fg">Tags</h2>`

It looks like the only style we want from `panel-title` is a `margin: 0 0 0.5em;`; we can also remove the `text-transform` override if we remove the `panel-title` class.

Reply:
I wondered and was going to ask if it would be ok to remove the `<PanelTitle>`, thanks for this. I've updated it accordingly.